beetmoverscript/constants.py

```diff
@@ -151,7 +151,8 @@
 )
 
 CHECKSUMS_CUSTOM_FILE_NAMING = {
-    'beetmover-source': '-source'
+    'beetmover-source': '-source',
+    'release-sign-and-push-langpacks': '-langpack',
```

Comment:
This is the culprit. This should be `release-beetmover-signed-langpacks` instead as the logic from https://github.com/mozilla-releng/beetmoverscript/pull/174/files#diff-21f3f939e3375901322bd3639f4394bcR93 looks into the ['tags']['kind'] which in this case is set to `release-beetmover-signed-langpacks`.